homeassistant/components/assist_satellite/entity.py

```diff
@@ -416,6 +416,7 @@ async def _resolve_announcement_media_id(
                 language=pipeline.tts_language,
                 options=tts_options,
             )
+            return AssistSatelliteAnnouncement(message, media_id, media_id_source)
```

Comment:
Seems like this has the potential to affect other assist satellite components, but the VOIP announcements wouldn't work otherwise.

Reply:
I added an internal_media_id property to the AssistSatelliteAnnouncement to hopefully mitigate this issue.